### ISSUE TITLE
[DOCS] Ability for Fiscal Hosts to "unhost" Collectives

### ISSUE THREAD

**Issue by shannondwray** [opencollective/opencollective#6136]
https://github.com/opencollective/opencollective/issues/3330 

## Context
 
Un-hosting collectives is a feature that required platform support as it was only possible via Root Actions. To remove the need to support action here, we are enabling an option for Host Admins to un-host their own collectives.

### What's changing?

The ability to un-host a collective was previous limited to the Root Actions menu of the platform. 
With this release, host admins will be able to un-host collectives from their host dashboard.

### Why are we changing it?

We are enabling host admins to take action and un-host a collective from their dashboard without need to contact support.

### Who does this change impact?

Host admins

### How do users use it?
 
Host admins can go to their admin dashboard and under each collect there is a set of actions that can be performed, one which is to un-host the collective.
The collective balance must be cleared before un-hosting, this can be done by processing expenses.

### Is this change optional or mandatory?

The option will be available to all hosts.

### Is this change being released incrementally as a beta? What's the schedule?

Immediate release

**Comment by shannondwray** [comment]
https://www.loom.com/share/4327241f9e924b43b26a24044a82febf

**Comment by shannondwray** [comment]
From @benjam's Release Draft on Open Collective - We've added the ability to freeze and un-host Collectives for host administrators.
 In order to protect hosts from abuse, and to support Collectives wishing (or needing) to migrate to another host we have added the ability for host admins to self-manage their relationships with their Collectives.



Freezing a Collective will block all financial activity within that Collective. Contributions and expenses will be removed from the Collective's public profile, and admins will be informed of why the action was taken. We've used this within our own host networks to stop fraudulent contributions and stop rogue administrators while we resolve situations with the communities impacted.

We've also added the ability to un-host a collective, which can be used to free a Collective to choose another host or to become an [independent collective](https://docs.opencollective.com/help/independent-collectives/about-independent-collectives). Before un-hosting, a Collective's [balance must be zeroed](https://docs.opencollective.com/help/collectives/collective-settings/zero-collective-balance) by paying expenses, making contributions to other Collectives or gifting the balance to the host.

Both of these options are available from your host dashboard under 'Hosted Collectives'.

📘 Read more about freezing a Collective
📙 Read more about un-hosting a Collective
✉️ Contact support
📣 Leave feedback below
